assets/scss/components/_summary-card.scss

```diff
@@ -54,7 +54,7 @@
     @extend .govuk-summary-card__title-wrapper;
     @include govuk-responsive-padding(0, "top");
     @include govuk-responsive-padding(0, "bottom");
-    @include govuk-responsive-margin(1, "left");
+    @include govuk-responsive-padding(4, "left");
```

Comment:
changed from margin to padding as there was a white border on the subheading when it had a grey background for overview page.